srcpkgs/stremio-shell/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'stremio-shell'
+pkgname=stremio-shell
+version=4.4.25
+revision=1
+_singleapplication_hash=6fbf6bffc848821d41a09ccd9fc0ede83bbbcdce
+build_style=qmake
+hostmakedepends="qt5-host-tools"
+makedepends="mpv-devel qt5-webview-devel qt5-webengine-devel
+ qt5-declarative-devel qt5-webchannel-devel qt5-location-devel
+ qt5-quickcontrols2-devel qt5-quickcontrols chromaprint-devel"
+depends="qt5-quickcontrols qt5-quickcontrols2 nodejs"
+short_desc="Hub for video content aggregation"
+maintainer="maxice8 <thinkabit.ukim@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://www.stremio.com"
+distfiles="https://github.com/Stremio/stremio-shell/archive/${version}.tar.gz
+ https://github.com/itay-grudev/SingleApplication/archive/${_singleapplication_hash}.tar.gz
+ https://dl.strem.io/four/v4.4.10/server.js>server.js.txt
```

Comment:
You can go with `${version}`
also `skip_extraction="server.js stremio.asar"` instead of `>*.txt`